src/Microsoft.Graph/Requests/Generated/SharedDriveItemRequest.cs

```diff
@@ -136,6 +137,30 @@ public ISharedDriveItemRequest Expand(string value)
         }
 
         /// <summary>
+        /// Adds the specified expand value to the request.
+        /// </summary>
+        /// <param name="expandExpression">The expression from which to calculate the expand value.</param>
+        /// <returns>The request object to send.</returns>
+        public ISharedDriveItemRequest Expand(Expression<Func<SharedDriveItem, object>> expandExpression)
```

Comment:
I saw you added several expand and select function there, as they are pretty same on the checking and extract logic, would it be possible to put these into a helper function instead of making duplicates everywhere?